src/parsing_routines_matrixnetworks.jl

```diff
@@ -122,3 +122,106 @@ function loadrxnetwork(ft::MatrixNetwork,
 
     ParsedReactionNetwork(rn, nothing)
 end
+
+
+"""
+    For loading networks with complex stoichiomatix ,incidence matrix.
+
+Notes:
+- Assumed that column of complex stoichiomatrix represents composition of reaction complexes,
+  with positive entries of size num_of_species x num_of_complexes, where
+  the non-zero positive entries in the kth column denote stoichiometric
+  coefficients of the species participating in the kth reaction complex.
+
+- The complex incidence matrix, is number of complexes by number of reactions with
+  Bᵢⱼ = -1, if the i'th complex is the substrate of the j'th reaction,
+         1, if the i'th complex is the product of the j'th reaction,
+         0, otherwise
+"""
+"""
```

Comment:
Try using a comment instead
```julia
# for dense matrices
```
and make the same change below for the spare version.

Reply:
resolved!